src/main/proto/ga4gh/rna_quantification.proto

```diff
@@ -70,25 +86,30 @@ message ExpressionLevel {
   // Name
   string name = 2;
 
-  // The associated QuantificationGroup. 
-  string quantification_group_id = 3;
+  // Id of the feature
+  string feature_id = 3;
+
+  // The associated ExpressionLevelGroup. 
+  string expression_level_group_id = 4;
```

Comment:
In tandem with my suggestion to rename `ExpressionLevelGroup` -> `FeatureGroup`, also move `string expression_level_group_id` to `repeated string feature_group_ids`, so that this expression level can participate in zero or more of such.